kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/BaseConnectorIT.java

```diff
@@ -49,22 +53,19 @@
   private static final long CONNECTOR_STARTUP_DURATION_MS = TimeUnit.MINUTES.toMillis(1);
   private static final long S3_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(30);
 
+  protected static AmazonS3 S3Client;
+  protected static final String TEST_BUCKET_NAME =
+      "connect-s3-integration-testing-" + System.currentTimeMillis();
   protected EmbeddedConnectCluster connect;
   protected Map<String, String> props;
 
-  @Before
-  public void setup() {
-    startConnect();
-    setupProperties();
-  }
```

Comment:
Let's keep the `setup()` here with `startConnect()`, and move `setupProperties()` under the before in `S3SinkConnectorIT`